tools/depmod.c

```diff
@@ -3043,17 +3043,17 @@ static int do_depmod(int argc, char *argv[])
 	cfg.dirnamelen = snprintf(cfg.dirname, PATH_MAX, "%s%s/%s", root ?: "",
 				  module_directory, cfg.kversion);
 	if (cfg.dirnamelen >= PATH_MAX) {
-		ERR("Bad directory %s" MODULE_DIRECTORY "/%s: path too long\n",
-		    root ?: "", cfg.kversion);
+		ERR("Bad directory %s%s/%s: path too long\n", root ?: "",
+		    module_directory, cfg.kversion);
```

Comment:
Let's add `Fixes: 4f8a6a85 ("tools: Check snprintf return value")` to the commit message.